pyfar/constants/constants.py

```diff
@@ -0,0 +1,44 @@
+"""Constant calculation."""
+import numpy as np
+
+
+def saturation_vapor_pressure_magnus(temperature):
+    r"""
+    Calculate the saturation vapor pressure of water in Pa using the
```

Comment:
Align between constants PRs?
```suggestion
    Calculate the saturation vapor pressure of water in Pascal using the
```